code/modules/mapfluff/ruins/spaceruin_code/transit_booth.dm

```diff
@@ -0,0 +1,7 @@
+/////////// transit_booth area
+/area/ruin/space/has_grav/transit_booth
+	name = "transit_booth"
+	icon = 'icons/area/areas_ruins.dmi'
+	icon_state = "ruins"
+	requires_power = FALSE
+	ambientsounds = list('sound/ambience/ambigen13.ogg','sound/ambience/ambigen14.ogg','sound/ambience/ambinice.ogg')
```

Comment:
You can put this in `code/game/area/areas/ruins/space.dm`, just tack it on the end. This directory would be used for custom objects in the ruin.